modules/bvl_feedback/test/TestPlan.md

```diff
@@ -1,10 +1,17 @@
 # Behavioural Feedback Test Plan
 
-1. Behavioural feedback button (notepad in the toolbar) should show up on the following pages:
- * Candidate Profile
- * Instrument List
- * Any instrument
-[Automation Test]
+1. Behavioural feedback button should show up on the following pages:
+ * Candidate Profile  
+ To find the Candiadate profile, first navigate to 'Access Profile', then select the specific Candidate profile from the list.  
```

Comment:
This edit is not necessarily unless you'd like to change it to tell the tester _where_ to look for feedback within each of these modules/pages (as with the pencil icon).  
It is very safe to assume that anyone testing Loris knows how to get to a candidate profile and instrument list page.  I see a few inaccuracies below, i'll correct as I go ...